### PR TITLE
Add unfancy@imbored mod

### PR DESCRIPTION
**Pull request by unfanciousthewizard**
a mod i am making, not much to say as it's pretty much as you see it

### CODE REVIEW

**Comment by unfanciousthewizard** [comment]
Fixed that just now

**Review by skyline69 — APPROVED** [review]
Looks good. downloadURL resolves to a real downloadable file and metadata is valid.

**Review by skyline69 — CHANGES_REQUESTED** [review]
Please change thumbnail file to .jpg. This PR currently uses thumbnail.png, and we only accept .jpg thumbnails.

**Comment by pawPatoes** [comment]
heres the thumbnail just remove the 1 from the name
thumbnail1

**Comment by unfanciousthewizard** [comment]
ok i fixed it

**Comment by pawPatoes** [comment]
now its a jpg, now we gotta wait for skyline to see it

**Comment by pawPatoes** [comment]
DUDE IM SO SORRY BUT PLEASE, DONT HARDCODE VALUES!!!
heres an example
SMODS.Joker {
    loc_txt = {
        text = {"gives #1# but can gain #2# and lose #3#"}}
    config = { extra = { x_mult = 1, x_chips = 1, gain = 2, loss = 1 } },
    loc_vars = function(self, info_queue, card) 
        return { vars = { 
            card.ability.extra.x_mult, 
            card.ability.extra.gain, 
            card.ability.extra.loss, 
            card.ability.extra.x_chips 
        } } 
    end,
so since we returned var card.ability.extra.x_mult first in the desc use #1# and keep going so #2# for gain ect.. for probablites use 
SMODS.Joker {
loc_txt = {
        text = {"has a #1# in #2# chance to explode every blind"}}
    config = { extra = { chance = 50 } }, 
    loc_vars = function(self, info_queue, card) 
        return { vars = { G.GAME.probabilities.normal or 1, card.ability.extra.chance } }
    end,
to score simply do
scoring smth = card.ability.extra.name_of_scoring
example:
Xmult = card.ability.extra.x_mult (do note that Xmult makes an automatic message but Xmult_mod doesnt add the message, you can remove messages with remove_default_message = true inside the return func)
chips and chips_mod also work
using these makes it work with oops all 6's and makes changing values exponentially easier

**Review by theminizilla — APPROVED** [review]
your fully good to go, except image file please change to suported image file

**Review by theminizilla — APPROVED** [review]

**Review by pawPatoes — APPROVED** [review]
Good to go, should be merged
